frontend/src/components/customer/ProfileDetailsCard.tsx/ProfileDetailsCard.tsx

```diff
@@ -0,0 +1,480 @@
+                                                                AdapterDayjs
+                                                            }
+                                                        >
+                                                            <DateField
+                                                                label="Date Of Birth"
+                                                                InputLabelProps={{
+                                                                    sx: {
+                                                                        fontFamily:
+                                                                            "Lato",
+                                                                        fontSize:
+                                                                            "1rem",
+                                                                        fontWeight:
+                                                                            "400",
+                                                                    },
+                                                                }}
+                                                                value={value}
+                                                                onChange={(
+                                                                    date
+                                                                ) => {
+                                                                    setValue(
+                                                                        date
+                                                                    );
+                                                                    field.onChange(
+                                                                        date
+                                                                    );
+                                                                }}
+                                                                onError={(
+                                                                    newError
+                                                                ) =>
+                                                                    setError(
+                                                                        newError
+                                                                    )
+                                                                }
+                                                                slotProps={{
+                                                                    textField: {
+                                                                        helperText:
+                                                                            errorMessage,
+                                                                    },
```

Comment:
Create a resuable DateField components in `components/common/form` directory so that it can be something reusable. Let me know if u face any struggles with this